python/GafferUI/GraphBookmarksUI.py

```diff
@@ -93,22 +93,22 @@ def appendPlugContextMenuDefinitions( nodeGraph, plug, menuDefinition ) :
 				}
 			)
 
-##########################################################################
-# Internal implementation
-##########################################################################
-
-def __getBookmarked( node ) :
+def getBookmarked( node ) :
 
 	return Gaffer.Metadata.value( node, "graphBookmarks:bookmarked" ) or False
 
-def __setBookmarked( node, bookmarked ) :
+def setBookmarked( node, bookmarked ) :
```

Comment:
I'd be fine with that, but would we still call it "graphBookmarks:bookmarked" at that point? Or something more general?